src/__tests__/reduxForm.spec.js

```diff
@@ -185,25 +185,25 @@ const describeReduxForm = (name, structure, combineReducers, expect) => {
       const spy = createSpy()
       const { dispatch } = propChecker({}, spy, {
         validate: values => {
-          const animal = getIn(values, 'animal')
-          return animal && animal.length > 5 ? { animal: 'Too long' } : {}
+          const foo = getIn(values, 'foo')
+          return foo && foo.length > 5 ? { foo: 'Too long' } : {}
```

Comment:
It's not arbitrary - the validation now uses `registeredFields` to determine which fields are required to be validated, and since `foo` is what's mounted in the fixture, not `animals`, it was not validating the field.

I had to change the name to match.